index.html

```diff
@@ -44,8 +44,138 @@ <h6><a href="">LET'S CONNECT</a></h6>
+                    <span class = "subheading-list-title">Back End Dev</span>
+                    <img src = "Images/dot-symbol.svg" alt = "choice-symbol" />
+                    <span class = "subheading-list-title">2015</span>
+                </div>
+                <article class = "card-content">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </article> 
+                <ul class = "web-languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button class = "see-project">See project</button>
+            </div>
+        </div>
+    </section>
+
+
+
+    <section class = "works">
+        <div class = "cards">
+                <img src = "Images/stories-2.png" alt = "about-professional"/>
+            <div class = "cards-description">
+                <h2 class = "cards-titles">Multi-Post
+                    Stories</h2>
+                <div class = "subheading-list">
+                    <span class = "canopy">CANOPY</span>
+                    <img src = "Images/dot-symbol.svg" alt = "choice-symbol"/>
+                    <span class = "subheading-list-title">Back End Dev</span>
+                    <img src = "Images/dot-symbol.svg" alt = "choice-symbol" />
+                    <span class = "subheading-list-title">2015</span>
+                </div>
+                <article class = "card-content">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </article> 
+                <ul class = "webdev-languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button class = "see-project">see project</button>
+            </div>
+        </div>
+    </section>
+
+
+    <section class="class about-myself">
+        <div>
+            <h1 class = "name-tag">About Myself</h1>
+            <p class = "bio-info">I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you 
+                like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+        </div>
+        <h6><a href="">LET'S CONNECT</a></h6>
+        <ul class = "unordered-list d-flex flex-row p-0">
+            <li><a href="https://twitter.com/Chandu1111112/media"><img src="Images/twitter.png" alt="twitter-icon" class = "icon-size"></a></li>
+            <li><a href="https://www.linkedin.com/in/prem-chand-maddati-15a69a1a8/"><img src="Images/linkdin-icon.png" alt="linkdin-icon" class = "icon-size"></a></li>
+            <li><a href=""><img src="Images/Medium.png" alt="Medium-icon" class = "icon-size"></a></li>
```

Comment:
- [ ] In this section, the content should be aligned with `flexbox`, because it offers a flexible way to layout content and it is responsive 📱 💻

- [ ] You may achieve that by just taking the `<div class="cardback2">` ... `</div>` out from the `<section class="Workssection"> `and replace the div tag into section tag.